website/docs/guides/relay-resolvers/context.md

```diff
@@ -0,0 +1,120 @@
+---
+id: context
+title: "Context"
+slug: /guides/relay-resolvers/context
+description: Context in Relay Resolvers
+---
+import {FbInternalOnly, fbContent} from 'docusaurus-plugin-internaldocs-fb/internal';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+In order to pass a service, or other values to be shared with all resolvers, the `LiveResolverStore` provides a means of passing context. This gets passed to the third argument of all resolvers (live and non-live).
```

Comment:
Maybe we could call out that this is analogous to the context argument used on the server which usually holds things like the database connection: https://graphql.org/learn/execution/#root-fields--resolvers